hydroshare/settings.py

```diff
@@ -290,7 +278,6 @@
     "mezzanine.galleries",
     "crispy_forms",
     "mezzanine.accounts",
-    "mezzanine.mobile",
     "haystack",
```

Comment:
@sblack-usu not sure whether removing "mezzanine.mobile" would cause any issues with styling of HS site on mobile devices, most likely not, but it'd be good if @Maurier can confirm here.

Reply:
it was deprecated and has been removed. https://github.com/stephenmcd/mezzanine/commit/f75cd3ce6f58ee5e51e6a02336237381efe1a65e#diff-cb86fa52440c3b29e9ad70ce1a82fe14